src/main/java/chess/Board.java

```diff
@@ -0,0 +1,20 @@
+package chess;
+
+import chess.pieces.Pawn;
+import java.util.ArrayList;
+
+public class Board {
+    private ArrayList<Pawn> pawns = new ArrayList<>();
```

Comment:
타입을 `List`로 해 주세요. 
앞으로 코드를 작성하실 때, 구체적인 구현 클래스보단 인터페이스에 의존하는 방향의 코드를 작성하고자 노력해보면 좋겠어요. 

https://medium.com/better-programming/code-against-interfaces-not-implementations-37b30e7ab992 

아직 어려운 글일 수 있는데 읽어보시는 것도 추천해요. 

지금 단계에서 이해하기 쉽지 않은 개념이겠지만, 학습 초반부에 잘못된 습관이 들 수 있어 리뷰합니다. 

```suggestion
    private List<Pawn> pawns = new ArrayList<>();

Reply:
감사합니다! 보내주신 링크도 읽어보았는데 지금은 명확히 이해가 되진 않지만 나중에 한번 더 읽어보겠습니다~